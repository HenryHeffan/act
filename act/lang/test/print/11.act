defproc testme (chan(int) X; chan(bool) Y) {
  int<1> a;
  bool b;
  
  chp {
    X?+a; X?-; X?+int(b); Y?bool(a); Y?b
  }
}
testme t;

