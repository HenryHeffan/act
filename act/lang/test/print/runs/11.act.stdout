defproc testme (chan(int<32>) X; chan(bool) Y);

defproc testme (chan(int<32>) X; chan(bool) Y)
{

/* instances */
bool b;
int<1> a;

/* connections */
chp {
X?+a;X?-;X?+int(b);Y?bool(a);Y?b
}
}


/* instances */
testme t;

/* connections */
